src/utils/BtnMsg.ts

```diff
@@ -41,21 +42,23 @@ export class BtnMsg extends EmitterBase
      * Contains convenience methods for easier creation of messages with attached buttons  
      * Use `.on("press")` to listen for button presses
      * @param message The message or reply content
-     * @param buttons Up to 5 ButtonBuilder instances - customIDs will be managed by this BtnMsg
+     * @param buttons Up to 5 rows of 5 ButtonBuilder instances - customIDs will be managed by this BtnMsg
      */
-    constructor(message: string | EmbedBuilder | EmbedBuilder[], buttons: ButtonBuilder | ButtonBuilder[], options?: Partial<BtnMsgOpts>)
+    constructor(message: string | EmbedBuilder | EmbedBuilder[], buttons: ButtonBuilder | Tuple<Tuple<ButtonBuilder, 1|2|3|4|5>, 1|2|3|4|5>, options?: Partial<BtnMsgOpts>)
```

Comment:
would've called it `buttonRows` to make it more clear but ig the jsdoc comment explains it

Reply:
:low_quality_pepe: